website/content/partials/telemetry-metrics/vault/rollback/attempt/mountpoint.mdx

```diff
@@ -2,4 +2,10 @@
 
 Metric type | Value | Description
 ----------- | ----- | -----------
-summary     | ms    | Time required to perform a rollback operation on the given mount point
\ No newline at end of file
+summary     | ms    | Time required to perform a rollback operation on the given mount point
+
+### vault.rollback.attempt ((#vault-rollback-attempt))
```

Comment:
The new metrics should have their own partials and the relevant metric indexes should be updated to include them.